src/node_url.cc

```diff
@@ -131,11 +131,13 @@ namespace url {
   static int ToUnicode(std::string* input, std::string* output) {
     output->reserve(input->length());
     *output = input->c_str();
```

Comment:
I actually was going to do this, but `*output = *input` is not equivalent to `*output = input->c_str()` if the string contains null bytes so I decided to leave it alone. Seems like that is not a concern so I'll go ahead and change it.